flaml/autogen/agentchat/groupchat.py

```diff
@@ -0,0 +1,177 @@
+import sys
+from typing import Dict, List, Optional, Union
+import random
+from .agent import Agent
+from .responsive_agent import ResponsiveAgent
+
+
+class BroadcastAgent(ResponsiveAgent):
```

Comment:
I really like this design. I only have some lint comments here. Feel free to ignore them. 


1. Maybe we can rename the "BroadcastAgent" as a "Room" or "AgentGroup" or something simpler. In the real world, participants are often in the same "room" (or Chat Group), and the message a participant said should be sent to everyone inside the room.  So, it contains the "agents" list, which is more intuitive.
2. Maybe, we can add another class "RoundRobinChatManager(ChatManagerAgent)" which asks each participant to speak one-by-one in a round fashion.

Reply:
renamed